tomviz/MainWindow.cxx

```diff
@@ -382,6 +382,21 @@ QString MainWindow::readScript(const QString &scriptName)
     QByteArray array = file.readAll();
     return QString(array);
   }
+// On OSX the above doesn't work in a build tree.  It is fine
+// for superbuilds, but the following is needed in the build tree
+// since the executable is three levels down in bin/tomviz.app/Contents/MacOS/
+#if __APPLE__
+  else
+  {
+  path = QApplication::applicationDirPath() + "/../../../../share/tomviz/scripts/";
```

Comment:
To be clear, this is a debugging print statement using Qt's debugging macros, so when you run the program it should print a path.  If you are building in release you may have to change the first part to `qWarning()` to get it to show anything.